core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -68,96 +68,6 @@
   public FileInfo() {}
 
   /**
-<<<<<<< HEAD
-   * Creates a new instance of {@link FileInfo} from thrift representation.
-   *
-   * @param fileInfo the thrift representation of a file information
-   */
-  protected FileInfo(alluxio.thrift.FileInfo fileInfo) {
-    mFileId = fileInfo.getFileId();
-    mName = fileInfo.getName();
-    mPath = fileInfo.getPath();
-    mUfsPath = fileInfo.getUfsPath();
-    mLength = fileInfo.getLength();
-    mBlockSizeBytes = fileInfo.getBlockSizeBytes();
-    mCreationTimeMs = fileInfo.getCreationTimeMs();
-    mCompleted = fileInfo.isCompleted();
-    mFolder = fileInfo.isFolder();
-    mPinned = fileInfo.isPinned();
-    mCacheable = fileInfo.isCacheable();
-    mPersisted = fileInfo.isPersisted();
-    mBlockIds = new ArrayList<>(fileInfo.getBlockIds());
-    mInMemoryPercentage = fileInfo.getInMemoryPercentage();
-    mLastModificationTimeMs = fileInfo.getLastModificationTimeMs();
-    mTtl = fileInfo.getTtl();
-    mTtlAction = ThriftUtils.fromThrift(fileInfo.getTtlAction());
-    mOwner = fileInfo.getOwner();
-    mGroup = fileInfo.getGroup();
-    mMode = fileInfo.getMode();
-    mPersistenceState = fileInfo.getPersistenceState();
-    mMountPoint = fileInfo.isMountPoint();
-    mFileBlockInfos = new ArrayList<>();
-    if (fileInfo.getFileBlockInfos() != null) {
-      for (alluxio.thrift.FileBlockInfo fileBlockInfo : fileInfo.getFileBlockInfos()) {
-        mFileBlockInfos.add(new FileBlockInfo(fileBlockInfo));
-      }
-    }
-    mMountId = fileInfo.getMountId();
-    mInAlluxioPercentage = fileInfo.getInAlluxioPercentage();
-    if (fileInfo.isSetUfsFingerprint()) {
-      mUfsFingerprint = fileInfo.getUfsFingerprint();
-    }
-    if (fileInfo.isSetAcl()) {
```

Comment:
where is this functionality now?

Reply:
I missed it in the first commit, so I added it in a subsequent commit. see  (https://github.com/Alluxio/alluxio/pull/7355/commits/7699db519d5ec42fef3c19945081a7887243ed67)